src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,111 @@
+.clock {
+  position: relative;
+
+  width: 80vmin;
+  height: 80vmin;
+
+  background-color: $color-bg;
+  border-radius: 50%;
+  border: 15px solid $color-line;
+
+  &__inner {
+    width: 100%;
+    height: 100%;
+
+    border-radius: 50%;
+    border: 10px solid $color-bg-inner;
+  }
+
+  &__fifteen {
+    position: absolute;
+
+    width: 3px;
+    height: 20px;
+
+    background-color: $color-line;
+
+    &--0 {
+      top: 10px;
+      left: 50%;
+
+      transform: translateX(-50%);
+    }
+
+    &--15 {
+      top: 50%;
+      right: 17px;
+
+      transform: translateY(-50%) rotate(90deg);
+    }
+
+    &--30 {
+      bottom: 10px;
+      left: 50%;
+
+      transform: translateX(-50%);
+    }
+
+    &--45 {
+      top: 50%;
+      left: 17px;
+
+      transform: translateY(-50%) rotate(90deg);
+    }
+  }
+
+  &__center {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+
+    transform: translate(-50%, -50%);
+
+    width: 50px;
+    height: 50px;
+
+    background-color: $color-line;
+    border-radius: 50%;
+
+    z-index: 3;
+  }
+
+  &__min {
+    position: absolute;
+    top: 20%;
+    left: calc(50% - 2px);
+
+    transform-origin: bottom center;
+
+    width: 4px;
+    height: 30%;
+
+    background-color: $color-line;
+
+    z-index: 1;
+
+    animation: time 3600s infinite steps(60);
```

Comment:
it will be better to use other timing for demo but it's not an issue
![image](https://github.com/mate-academy/layout_stop-watch/assets/22824431/aa570a45-b9af-4412-8a3a-fafbd9369dfa)